token/src/instruction.rs

```diff
@@ -24,64 +29,121 @@ pub struct TokenInfo {
 pub enum TokenInstruction {
     /// Initializes a new mint and optionally deposits all the newly minted tokens in an account.
     ///
-    /// # Accounts expected by this instruction:
+    /// * Accounts expected by this instruction:
     ///
-    ///   0. `[writable, signer]` New mint to create.
+    ///   0. `[writable, signer]` The mint to initialize.
     ///   1.
-    ///      * If supply is non-zero: `[writable]` Account to hold all the newly minted tokens.
-    ///      * If supply is zero: `[]` Owner of the mint.
-    ///   2. Optional: `[]` Owner of the mint if supply is non-zero, if present then further
-    ///      minting is supported.
+    ///      * If supply is non-zero: `[writable]` The account to hold all the newly minted tokens.
+    ///      * If supply is zero: `[]` The owner/multisignature of the mint.
+    ///   2. `[]` (optional) The owner/multisignature of the mint if supply is non-zero, if
+    ///                      present then further minting is supported.
+    ///
     InitializeMint(TokenInfo),
-    /// Initializes a new account.
+    /// Initializes a new account to hold tokens.
     ///
-    /// # Accounts expected by this instruction:
+    /// * Accounts expected by this instruction:
     ///
-    ///   0. `[writable, signer]`  New account being initialized.
+    ///   0. `[writable, signer]`  The account to initialize.
     ///   1. `[]` The mint this account will be associated with.
-    ///   2. `[]` Owner of the new account.
+    ///   2. `[]` The new account's owner/multisignature.
     InitializeAccount,
+    /// Initializes a multisignature account with N provided signers.
+    ///
+    /// Multisignature accounts can used in place of any single owner/delegate accounts in any
+    /// token instruction that require an owner/delegate to be present.  The variant field represents the
+    /// number of signers (M) required to validate this multisignature account.
+    ///
+    /// * Accounts expected by this instruction:
+    ///
+    ///   0. `[signer, writable]` The multisignature account to initialize.
+    ///   1. ..1+N. `[]` The signer accounts, must equal to N where 1 <= N <= 11.
+    InitializeMultisig(u8),
     /// Transfers tokens from one account to another either directly or via a delegate.
     ///
-    /// # Accounts expected by this instruction:
+    /// * Accounts expected by this instruction:
+    ///
+    ///   * Single owner/delegate
+    ///
+    ///   0. `[writable]` The source account.
+    ///   1. `[writable]` The destination account.
+    ///   2. '[signer]' The source account's owner/delegate.
```

Comment:
Will this turn out ok in markdown?  I find it a little more readable than the existing comment
```suggestion
    ///   Single owner/delegate:
    ///      0. `[writable]` The source account.
    ///      1. `[writable]` The destination account.
    ///      2. '[signer]' The source account's owner/delegate.
```

Reply:
Nope, wraps "0..." up on top of the line before.